en/docs/key-concepts/identity-provisioning-and-its-standards.md

```diff
@@ -16,8 +16,6 @@ the following diagram.
 
 ![User and identity provisioning using WSO2 Identity Server](../../assets/img/concepts/identity-provisioning.png)
```

Comment:
the image paths should start with "/assets/img..." instead of "../../assets/img"